cmd/virsnap/create.go

```diff
@@ -2,47 +2,44 @@
 // Licensed under the MIT License. You have obtained a copy of the License at
 // the "LICENSE" file in this repository.
 
-// Package cmd implements the handlers for the different command line arguments.
```

Comment:
Re-add, please.

Reply:
It's not necessary to include the same package level documentation comment in every file, since it's package scoped it's enough to add it once in a single file (for example in `clean.go`) and godoc will parse it for the whole package. Alternatively we can add a `doc.go` with a bigger explanation about the code. See [Godoc: documenting Go Code](https://blog.golang.org/godoc-documenting-go-code) for more details.